### PR TITLE
Drop public access to remaining members of pcmk_scheduler_t

### PR DESCRIPTION
**Pull request by kgaillot**

### CODE REVIEW

**Comment by kgaillot** [comment]
@nrwahl2 , glad to be done with this :)

File: lib/pengine/pe_actions.c [hunk 1]
```diff
@@ -1203,7 +1203,7 @@ node_priority_fencing_delay(const pcmk_node_t *node,
     GList *gIter = NULL;
 
     // PCMK_OPT_PRIORITY_FENCING_DELAY is disabled
-    if (scheduler->priority_fencing_delay <= 0) {
+    if (scheduler->priv->priority_fencing_ms <= 0U) {
```
Comment (nrwahl2) [inline lib/pengine/pe_actions.c:1206]: could never be `< 0` but no need to change now

**Review by nrwahl2 — APPROVED** [review]
Hooray!

File: lib/pengine/pe_actions.c (hunk 1 above)
Comment (kgaillot) [reply lib/pengine/pe_actions.c]: actually static analysis sometimes complains about that, updated commit

**Review by nrwahl2 — APPROVED** [review]

**Pull request merged**